.github/actions/run_local_pytest/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Run Local Pytest'
+description: 'Installs Python dependencies and runs Pytest against locally built ArcticDB binaries'
+inputs:
+  job_type:          {default: '', type: string, description: Specifies which steps should be executed}
```

Comment:
Just curious: is this ever not `run`?

Reply:
Yeah in the [benchmarks job](https://github.com/man-group/ArcticDB/blob/941-self-hosted-runners-for-ci/.github/workflows/analysis_workflow.yml#L229) because the "run" part is done by ASV

Reply:
When the `job_type` is not `run`, this action only does two lines of bash (after our simplification).

It then becomes a bit misleading to call this action, since the caller don't actually want to 'Run Local Pytest'